cmake/setup_lib_find_paths.cmake

```diff
@@ -38,16 +38,19 @@
 # actual libraries are found in the CMake files of the individual componenents.
 #------------------------------------------------------------------------------
 
-#------------------------------------------------------------------------------
-# emit warning if CMAKE_FIND_ROOT_PATH is used instead of CMAKE_PREFIX_PATH
-if(NOT "${CMAKE_FIND_ROOT_PATH}" STREQUAL "" AND "${CMAKE_PREFIX_PATH}" STREQUAL "")
-  set(CMAKE_PREFIX_PATH "${CMAKE_FIND_ROOT_PATH}")
-  message(STATUS "Please use CMAKE_PREFIX_PATH instead of CMAKE_FIND_ROOT_PATH ")
-endif()
 
 #------------------------------------------------------------------------------
-# add contrib location of source package <source>/contrib
-list(APPEND CMAKE_PREFIX_PATH ${OPENMS_HOST_DIRECTORY}/contrib/)
+# Specify the path to the contrib-build. Will be added to CMAKE_PREFIX_PATH
+# for searching and as first entry in the includes/libraries to avoid
+# mismatches with installed system libraries
+if(NOT OPENMS_CONTRIB_LIBS)
+  message(FATAL_ERROR "Please specify an explicit path to the built contrib libraries via
```

Comment:
I dont think making this a fatal error is a good idea - distros for example will not want this at all as libOpenMS should link only to system libraries! So I suggest to change this to a warning ...